python/ray/worker.py

```diff
@@ -1641,6 +1641,29 @@ def wait(object_ids, num_returns=1, timeout=None):
         return ready_ids, remaining_ids
 
 
+def kill(actor):
+    """Kill an actor forcefully.
+
+    This will interrupt any running tasks on the actor, causing them to fail
+    immediately. Any atexit handlers installed in the actor will still be run.
+
+    If you want to kill the actor but let pending tasks finish,
+    you can call ``actor.__ray_terminate__.remote()`` instead to queue a
+    termination task.
+
+    If this actor is reconstructable, it will be attempted to be reconstructed.
```

Comment:
This behavior sounds right to me. Just saw this PR which might be doing the opposite (though I didn't check) https://github.com/ray-project/ray/pull/7346.

Reply:
Can this be optional? Like an argument to `kill`? Or maybe `ray.restart(actor)` when you want to reconstruct it? I find it strange for this to happen automatically?